R/utils_clean_stan_parameters.R

```diff
@@ -25,7 +25,7 @@
 
 #' @keywords internal
 .clean_parameters <- function(x) {
-  removers <- grep("^(prior_|sd_|cor_|lp__)", x$Parameter)
+  removers <- grep("^(prior_|sd_|cor_|lp__|smooth_sd)", x$Parameter)
```

Comment:
Shouldn't we deal such name cleaning in insight?

Reply:
Yes, true. I'll open an issue there, and once we've implemented it in insight, we can remove the code from here...